tasks/configure-OpenBSD.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+# XXX rpc.lockd(8) of OpenBSD does not support client-side of the protocol
```

Comment:
if this comment means `# rpc.lockd is supposed to be configured here. However,`,  it should be written explicitly.